config/default.yml

```diff
@@ -3538,6 +3538,9 @@ Style/FetchEnvVar:
   VersionAdded: '1.28'
   # Environment variables to be excluded from the inspection.
   AllowedVars: []
+  # - `SecondArgOfFetch` puts the RHS to the second argument of `ENV.fetch`. (default)
+  # - `BlockContent` puts the RHS into a block.
+  BasicLiteralRhs: 'SecondArgOfFetch'
```

Comment:
Seems to me that adds extra complexity for the end users - I generally don't see why someone would want to put basic literals in a block, as there are no performance gains for this.

Reply:
That's true. I removed the option. https://github.com/rubocop/rubocop/pull/10585/commits/37979149a76054a4879fc8121f74ab92c33bc1a3